lib/vizkit/vizkit_items.rb

```diff
@@ -803,6 +803,27 @@ def write(&block)
         end
     end
 
+    class TaskModelItem < VizkitItem
+        attr_reader :model
+
+        def initialize(task, options = Hash.new)
```

Comment:
Sure, I don't use it, but then how is not caching options maintaining the API, since it is an attribute on VizkitItem?